src/cascadia/WpfTerminalControl/TerminalContainer.cs

```diff
@@ -161,6 +164,13 @@ internal string GetSelectedText()
         /// <param name="renderSize">Size of the rendering window.</param>
         internal void Resize(Size renderSize)
         {
+            if (renderSize.Width == 0 || renderSize.Height == 0)
+            {
+                throw new ArgumentException(nameof(renderSize), "Terminal column or row count cannot be 0.");
+            }
+
+            var dpiScale = VisualTreeHelper.GetDpi(this);

```

Comment:
i don't see why we need this line..

Reply:
Looks like it came from a bad merge. Pushing a fix...